samples/bluetooth/direction_finding_peripheral/sample.yaml

```diff
@@ -2,22 +2,66 @@ sample:
   name: Direction Finding Peripheral
   description: Sample application showing peripheral role of Direction Finding in connected mode
 tests:
-  sample.bluetooth.direction_finding_peripheral_nrf:
+  sample.bluetooth.direction_finding_peripheral.aoa.sdc.nrf52_nrf53:
     sysbuild: true
+    extra_args:
+      - OVERLAY_CONFIG="overlay-aoa.conf"
     build_only: true
-    platform_allow: nrf52833dk/nrf52833 nrf52833dk/nrf52820 nrf5340dk/nrf5340/cpuapp
+    platform_allow:
+      - nrf52833dk/nrf52833
+      - nrf52833dk/nrf52820
+      - nrf5340dk/nrf5340/cpuapp
     tags: bluetooth sysbuild
     integration_platforms:
       - nrf52833dk/nrf52833
       - nrf52833dk/nrf52820
       - nrf5340dk/nrf5340/cpuapp
-  sample.bluetooth.direction_finding_peripheral_nrf.aod:
+  sample.bluetooth.direction_finding_peripheral.aoa.bt_ll_sw_split.nrf52:
+    sysbuild: false
+    build_only: true
+    extra_args:
+      - OVERLAY_CONFIG="overlay-aoa.conf;overlay-bt_ll_sw_split.conf"
+      - SNIPPET="bt-ll-sw-split"
+    platform_allow:
+      - nrf52833dk/nrf52833
+      - nrf52833dk/nrf52820
+    tags: bluetooth
+    integration_platforms:
+      - nrf52833dk/nrf52833
+      - nrf52833dk/nrf52820
+  sample.bluetooth.direction_finding_peripheral.aoa.bt_ll_sw_split.nrf53:
     sysbuild: true
-    extra_args: OVERLAY_CONFIG="overlay-aoa.conf"
     build_only: true
-    platform_allow: nrf52833dk/nrf52833 nrf52833dk/nrf52820 nrf5340dk/nrf5340/cpuapp
+    extra_args:
+      - OVERLAY_CONFIG="overlay-aoa.conf"
+      - ipc_radio_OVERLAY_CONFIG="../../overlay-bt_ll_sw_split.conf"
+      - ipc_radio_SNIPPET="bt-ll-sw-split"
+    platform_allow:
+      - nrf5340dk/nrf5340/cpuapp
     tags: bluetooth sysbuild
+    integration_platforms:
+      - nrf5340dk/nrf5340/cpuapp
+  sample.bluetooth.direction_finding_peripheral.all.bt_ll_sw_split.nrf52:
+    sysbuild: false
+    build_only: true
+    extra_args:
+      - OVERLAY_CONFIG="overlay-bt_ll_sw_split.conf"
+      - SNIPPET="bt-ll-sw-split"
+    platform_allow:
+      - nrf52833dk/nrf52833
+      - nrf52833dk/nrf52820
+    tags: bluetooth
```

Comment:
AoA needs the overlay-aoa.conf.

Hence, two separate targets are required each for nrf52 and nrf53. Or am I misunderstanding something?

Reply:
talked offline with author... we believe peripheral will support both AoA and AoD, a separate build is not required.

Reply:
If AoD is defined, BT_DF_CTE_TYPE_ALL is used in the peripheral sample.
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/direction_finding_peripheral/src/main.c#L49